source/wp-content/themes/wporg-developer/scss/_colors.scss

```diff
@@ -0,0 +1,86 @@
+//Colors
```

Comment:
I added this file, which is from [the Core WP Colors](https://codepen.io/ryelle/full/WNGVEjw), but could probably remove it since I'm not using many colors from it after all. Any opinions?